EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc

```diff
@@ -37,6 +37,16 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf):
     fedUnpackList_.push_back(1118);
     fedUnpackList_.push_back(1120);
     fedUnpackList_.push_back(1122);
+    // HBHE uTCA
```

Comment:
making this inclusive loop will 
- prevent these unpacker changes (apparently forgotten until the last minute)
- get rid of the hardcoded FED numbers (FedNumbering is preferred)